index.html

```diff
@@ -50,8 +50,41 @@ <h1 class="lg-header lg-header-color">COMPLETE GOODNESS</h1>
         <a href="/Pages/ProductPage/product.html">
             <button class="button overlay-text button-overlay">SHOP NOW</button>
         </a>
+        <h1 class="cart-page-header">Featured Categories</h1>
+        <div class="featured-categories">
```

Comment:
for semantic purposes you can use a p tag with a higher font size cause these dont look like headings